File: Java/src/main/java/com/semantria/mapping/output/Facet.java
package com.semantria.mapping.output;

import javax.xml.bind.annotation.XmlElement;
import javax.xml.bind.annotation.XmlElementWrapper;
import java.util.List;

public class Facet 
{
	private String label = null;
	private Integer count = null;
	private Integer negative_count = null;
	private Integer positive_count = null;
	private Integer neutral_count = null;
	List<Attribute> attributes = null;
	List<Mention> mentions = null;
	
	@XmlElement(name="label")
	public String getLabel(){ return label; }
	@XmlElement(name="count")
	public Integer getCount(){ return count; }
	@XmlElement(name="negative_count")
	public Integer getNegativeCount(){ return negative_count; }
	@XmlElement(name="positive_count")
	public Integer getPositiveCount(){ return positive_count; }
	@XmlElement(name="neutral_count")
	public Integer getNeutralCount(){ return neutral_count; }
	@XmlElementWrapper(name="attributes")
	@XmlElement(name="attribute")
	public List<Attribute> getAttributes() { return attributes; }
	@XmlElementWrapper(name="mentions")
	@XmlElement(name="mention")
	public List<Mention> getMentions() { return mentions; }
	
	public void setLabel(String label){ this.label = label; }
	public void setCount(Integer count){ this.count = count; }
	public void setNegativeCount(Integer count){ this.negative_count = count; }
	public void setPositiveCount(Integer count){ this.positive_count = count; }
	public void setNeutralCount(Integer count){ this.neutral_count = count; }
	public void setAttributes(List<Attribute> attributes) { this.attributes = attributes; }
	public void setMentions(List<Mention> mentions) { this.mentions = mentions; }
}

File: dotNet/Semantria/Mapping/Output/Facet.cs
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
using System.Xml.Serialization;
using Semantria.Com.Mapping.Output;

namespace Semantria.Com.Mapping.Output
{
    [DataContract(Name = "facet")]
    [XmlRootAttribute("facet", Namespace = "")]
    public class Facet
    {
        [DataMember(Name = "label")]
        [XmlElementAttribute("label")]
        public string Label { get; set; }

        [DataMember(Name = "count")]
        [XmlElementAttribute("count")]
        public int Count { get; set; }

        [DataMember(Name = "negative_count")]
        [XmlElementAttribute("negative_count")]
        public int NegativeCount { get; set; }

        [DataMember(Name = "positive_count")]
        [XmlElementAttribute("positive_count")]
        public int PositiveCount { get; set; }

        [DataMember(Name = "neutral_count")]
        [XmlElementAttribute("neutral_count")]
        public int NeutralCount { get; set; }

        [DataMember(Name = "attributes")]
        [XmlArrayAttribute("attributes")]
        [XmlArrayItemAttribute("attribute", typeof(Attribute))]
        public List<Attribute> Attributes { get; set; }

        [DataMember(Name = "mentions")]
        [XmlArrayAttribute("mentions")]
        [XmlArrayItemAttribute("mention", typeof(CollMention))]
        public List<CollMention> Mentions { get; set; }
    }
}
